### ISSUE TITLE
[COGNITION]: Images of campus maps need semantic markup for screen reader usability

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#5987]
## Description

Some of the pages include a map of the health campuses. These maps are static images that do not provide enough contextual information to screen readers and assistive technology. By including some descriptive (semantic) markup, we can improve the experience for assistive tech users. Screenshot and code snippet included below.

## Type of Issue
Content

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

**Definition of done**
- [ ] Screen reader users can understand the relationship of buildings to each other, and/or streets
- [ ] Screen reader devices can read out the description list information correctly
- [ ] Markup validates as HTML5
- [ ] Markup shows no axe errors on future scans

## Possible Fixes (optional)

I mocked up a scenario using `<figure>`, `<figcaption>` and a description list. This has been tested for HTML5 valid markup and will offer screen readers several hooks. The heading level (H3, H4) should be adjusted to nest properly given the page structure.

```html
<figure>
  <img alt="University Drive campus map" src="/img/source/here.png">
  <figcaption>
    The University Drive campus has four main buildings. Building 1 - Main Hospital, Building 29 -
    Consolidation Building, Building 30 - Research Office Building, and Building 33 - Fisher House.
  </figcaption>
</figure>
<h3 id="facility-locations">University Drive Campus - Facility Map Descriptions</h3>
<dl aria-labelledby="facility-locations">
  <dt>Building 1 - Main Hospital</dt>
  <dd>South of Brackenridge Street. Main entrance is on the east end of the building, on Allequippa
  Street.</dd>
  <dt>Building 29 - Consolidation Building</dt>
  <dd>On the northeast corner of Building 1. Main entrance is on the southeast corner, on
  Allequippa Street.</dd>
</dl>
```

## WCAG or Vendor Guidance (optional)

* [Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)
* [MDN: The Description List](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl)
* [Caniuse: Figure and Semantic HTML5 Elements](https://caniuse.com/#search=figure)

## Screenshots or Trace Logs

![Screen Shot 2020-02-19 at 10 59 02 AM](https://user-images.githubusercontent.com/934879/74856616-019afe00-5308-11ea-9b12-fc1e0aa7428f.png)

**Comment by davidconlon** [comment]
Not sure if this is the right solution. I think that these images are supposed to be using a 508-c service for this.  I've pinged Jenny for more info.

**Comment by 1Copenut** [comment]
@davidconlon What's a 508-c service? I'm not familiar with this, but knowing their intent and what they'll bring to the page or the users can help us determine if this issue is still relevant.

**Comment by davidconlon** [comment]
Sorry - Medicaid lingo :-)   (508-c = 508 compliance)
Jenny mentioned that many of these maps are produced by a SaaS company that has already been vetted by the central 508 office for compliance. 

That being said this format doesn't look like one that is from that service so I'm wondering if this is an edge case of if we are using the wrong one.

**Comment by mmiddaugh** [comment]
Need to consider how best to support good mark-up when faciliy maps are updated in the future

**Comment by davidconlon** [comment]
@mmiddaugh @jenstrickland 

I think that this is a case where we need to think about what we want to do in order to teach this.

There will be ~150 of these campus maps _not maintained by DEPO_ but rather by the VAMCs themselves.
So how do we teach this to VAMC maintainers who are not web experts.

This is one of those areas that I've been talking about in terms of the alignment of our due dilligence for VA.gov but also the herding of proverbial cats.

Would love your thoughts on this.

**Comment by jenstrickland** [comment]
@davidconlon 

- I would love to review the training materials and any documented processes used for onboarding people (administrators, webmasters, and others) who will be adding content to various CMS pages. Applying an instructional design lens would help. Ideally we would provide quality guidance here.
- Additionally, can we evaluate the CMS admin interface to ensure we support users in the interface? Adding quality guidance meeting people where they are when they are adding content seems the most effective way to support them, and reinforce training (which is likely quickly forgotten). I'm not sure if the interface is designed and customizable.

**Comment by mmiddaugh** [comment]
Reviving this for training consideration @kevwalsh
